### PR TITLE
Add sequence id to batch response

### PR DESCRIPTION
**Pull request by samfrench**
Add sequence id to batch response

https://jira.dev.bbc.co.uk/browse/CONNPOL-3488

### CODE REVIEW

**Comment by samfrench** [comment]
@dazoakley, this PR adds the sequence id to batch responses.

File: lib/alephant/broker/response/batch.rb [hunk 1]
```diff
@@ -35,7 +35,9 @@ def json
               "status"       => component.status,
               "content_type" => component.content_type,
               "body"         => component.content
-            }
+            }.tap do |headers|
+              headers["seq_id"] = component.headers["X-Sequence"] if component.headers["X-Sequence"]
```
Comment (dazoakley) [inline lib/alephant/broker/response/batch.rb]: Just being a pedant, but could we use `sequence_id` or is there already something expecting `seq_id`?

Comment (samfrench) [inline lib/alephant/broker/response/batch.rb]: I have changed this to your suggestion. I thought it was good to keep it smaller, but don't mind.

Comment (dazoakley) [inline lib/alephant/broker/response/batch.rb]: I'd prefer being verbose if at all possible - just so there's no confusion

**Comment by dazoakley** [comment]
👍

**Pull request merged**